Executable/Injector.cs

```diff
@@ -1,25 +1,15 @@
-﻿using Mono.Cecil;
-using Mono.Cecil.Cil;
-using System;
+﻿using System;
 using System.IO;
-using System.Linq;
 
 namespace QModManager.Executable
 {
     internal class Injector
```

Comment:
@PrimeSonic we will need to talk about the injector and how we plan on moving things forward with this

Reply:
At this point, knowing that Mac and PC are using different installers anyways, I am not opposed to  the idea of using DoorStop for the injector on PC.  
If we can include a simple toggle to disable this on demand without having to go through an un-installation, that would be even better.

Reply:
That's what I was thinking: Having the injector located in the base game folder acting as a toggle for QModManager which basically copies/deletes the doorstop files from the game's folder.

Although I'm not sure if I want to have the mac executable and windows executable different. @legojoshua12 what would you prefer?

Reply:
Either that, or we could add a new entry in the steam library (and a desktop shortcut for epic games users) to run subnautica modded.

Reply:
I actually don't think having different installers is bad since its already that way, unless we can figure out how to combine the two into one installer.  Since it uses manual mono injection before, does that mean with two separate installers we would have one doorstop and the other on mono still?